docs/components/rendering_iterables.md

```diff
@@ -125,7 +184,7 @@ def projects_example() -> rx.Component:
     return rx.box(rx.foreach(NestedStateFE.projects, project_item))
 ```
 
-If you want an example where not all of the values in the dict are the same type then check out the example on [var operations using foreach]({vars.var_operations.path}).
+If you want an example where not all of the values in the dict are the same type then check out the example on [var operations using foreach]({docs.vars.var_operations.path}).
```

Comment:
I think this is so so important there should be an example here of using `rx.Base`. It is the biggest question we get again and again with dict with values of different types. This is something we must add in these docs, not just a link.

Reply:
we are addressing this elsewhere so I think we can ignore this comment for now